pkg/kubelet/types/types.go

```diff
@@ -85,6 +85,17 @@ func SortInitContainerStatuses(p *v1.Pod, statuses []v1.ContainerStatus) {
 	}
 }
 
+func SortedInitContainerStatuses(p *v1.Pod, statusMap map[string]*v1.ContainerStatus) []v1.ContainerStatus {
```

Comment:
If we do decide to move forward with this method, can we please use a different name than `SortedInitContainerStatuses`? Its confusing that we also have `SortedContainerStatuses` which is a custom type.

Additionally, could we please add a unit test for this function?

Reply:
Reiterating these requests because it looks like we're moving forward with this change :)

Reply:
Renamed to SortStatusesOfInitContainers